docs/src/vis.md

```diff
@@ -0,0 +1,82 @@
+# Visualization
+
+SparseMatrixColorings provides some internal utilities for visualization of matrix colorings via the un-exported function [`show_colors`](@ref).
+
+!!! warning
+    This function makes use of the [JuliaImages](https://juliaimages.org) ecosystem.
+    Using it requires loading at least [Colors.jl](https://github.com/JuliaGraphics/Colors.jl).
+    We recommend loading the full [Images.jl](https://github.com/JuliaImages/Images.jl) package for convenience, which includes Colors.jl.
+
+## Basic usage
+
+To obtain a visualization, simply call `show_colors` on a coloring result:
+
+```@example img
+using Images
+using SparseMatrixColorings, SparseArrays
+using SparseMatrixColorings: show_colors
+
+S = sparse([
+    0 0 1 1 0 1
+    1 0 0 0 1 0
+    0 1 0 0 1 0
+    0 1 1 0 0 0
+]);
+
+problem = ColoringProblem(; structure=:nonsymmetric, partition=:column)
+algo = GreedyColoringAlgorithm(; decompression=:direct)
+result = coloring(S, problem, algo)
+show_colors(result)
+```
+
+!!! tip "Terminal support"
+    Loading [ImageInTerminal.jl](https://github.com/JuliaImages/ImageInTerminal.jl) will allow you to show the output of `show_colors` within your terminal.
+    If you use VSCode's Julia REPL, the matrix will be displayed in the plot tab.
```

Comment:
I spend a solid minute wondering where it was ^^

Reply:
Interesting, somehow this didn't happen to me. Maybe because I don't use the VSCode REPL but launch my stuff manually via `julia --banner=no --project='.' -e 'TestEnv.activate()' -I`?